src/pages/validator/external-chains/centrifuge.mdx

```diff
@@ -37,27 +65,23 @@ Change the `ports` based on your network setup. Replace `/mnt/my_volume/data` wi
   services:
     centrifuge:
       container_name: centrifuge-chain
-      image: "centrifugeio/centrifuge-chain:main-[latest_release]"
+      image: "centrifugeio/centrifuge-chain:main-latest"
```

Comment:
is the `latest` image stable? For e.g. does it point to breaking versions before the chain itself is upgraded?

Reply:
It will be, we just pushed a big release for our new pipeline which includes a more standardize Docker tagging. Only official finalized releases will now publish to the `latest` tag, but I can't guarantee that for the existing latest and until we do the next release in a few weeks this won't be the case.
Let's not hold off this PR because of this though. It's easy to identify the tag that needs to be pushed given a certain release by looking at the SHA from the release commit